src/main/java/core/basesyntax/Colorable.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax;
+
+public interface Colorable {
+
+    void draw();
```

Comment:
The `draw()` method does not seem to be conceptually related to the `Colorable` interface. Consider separating concerns into different interfaces or classes.